### PR TITLE
Feature/reactstrap semantic refactor

### PR DESCRIPTION
**Pull request by Hammurobbie**
# Description

Fixes # (issue)

Removes all traces of bootstrap and reactstrap and refactors to Semantic UI. Also adds small "edit label" component.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by Cireimu — APPROVED** [review]

**Comment by lawsonarichard** [comment]
Since it hasn’t been touched for a while, I’d suggest pulling in master to the branch and double check everything functions and then review it again.

**Review by Cireimu — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "@material-ui/icons": "^4.5.1",
     "@okta/okta-react": "^1.3.1",
     "@okta/okta-signin-widget": "^3.4.2",
+    "@stardust-ui/docs-components": "^0.40.0",
     "@urql/exchange-graphcache": "^2.0.0",
     "axios": "^0.19.0",
     "bootstrap": "^4.4.1",
```
Comment (Memitaru) [inline package.json]: If we're not using bootstrap we need to remove this dependency as well

File: src/components/Settings/UpdateLabel.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  };
+
+  const [updateLabel, setUpdateLabel] = useState(initState);
+  const [, executeUpdate] = useMutation(updateLabelMutation);
+
+  const handleSubmit = useCallback(
+    e => {
+      e.preventDefault();
+      executeUpdate(updateLabel);
+      console.log(updateLabel);
```
Comment (Memitaru) [inline src/components/Settings/UpdateLabel.jsx]: Console log to remove

File: src/components/Settings/UpdateLabel.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  );
+
+  const handleChanges = e => {
+    e.preventDefault();
+    setUpdateLabel({
+      ...updateLabel,
+      [e.target.name]: e.target.value,
+    });
+  };
+  console.log('Label props', updateLabel);
```
Comment (Memitaru) [inline src/components/Settings/UpdateLabel.jsx]: Another console log

**Review by lawsonarichard — APPROVED** [review]
Tested and works locally!

**Pull request merged**